devel/example_notebooks/example1.ipynb

```diff
@@ -23,43 +23,9 @@
    "source": [
     "def prepare_recording_sorting():\n",
     "    # Simulate a recording (toy example)\n",
-    "    recording, sorting = se.example_datasets.toy_example(duration=50, num_channels=4, K=4, seed=1)\n",
-    "    # Write data to temporary files and store in a labbox-ephys-compatible format\n",
-    "    with hi.TemporaryDirectory() as tmpdir:\n",
-    "        raw_fname = tmpdir + '/raw.dat'\n",
-    "        traces: np.ndarray = recording.get_traces()\n",
-    "        traces = (traces / np.max(np.abs(traces)) * 10000).astype(np.int16)\n",
-    "        se.BinDatRecordingExtractor.write_recording(recording, raw_fname, time_axis=0, dtype=np.int16)\n",
-    "        bin_uri = kp.store_file(raw_fname)\n",
-    "        channel_ids = [int(id) for id in recording.get_channel_ids()]\n",
-    "        channel_map = dict()\n",
-    "        channel_positions = dict()\n",
-    "        locations = recording.get_channel_locations()\n",
-    "        for id, ii in enumerate(channel_ids):\n",
-    "            channel_map[str(id)] = ii\n",
-    "            channel_positions[str(id)] = locations[ii].tolist()\n",
-    "        recording_obj = {\n",
-    "            'recording_format': 'bin1',\n",
-    "            'data': {\n",
-    "                'raw': bin_uri,\n",
-    "                'raw_num_channels': recording.get_num_channels(),\n",
-    "                'num_frames': recording.get_num_frames(),\n",
-    "                'samplerate': recording.get_sampling_frequency(),\n",
-    "                'channel_ids': recording.get_channel_ids(),\n",
-    "                'channel_map': channel_map,\n",
-    "                'channel_positions': channel_positions\n",
-    "            }\n",
-    "        }\n",
-    "        R = le.LabboxEphysRecordingExtractor(recording_obj)\n",
-    "\n",
-    "        sorting_path = tmpdir + '/sorting.h5'\n",
-    "        le.extractors.H5SortingExtractorV1.write_sorting(sorting, sorting_path)\n",
-    "        S = le.LabboxEphysSortingExtractor({\n",
-    "            'sorting_format': 'h5_v1',\n",
-    "            'data': {\n",
-    "                'h5_path': kp.store_file(sorting_path)\n",
-    "            }\n",
-    "        })\n",
+    "    recording, sorting = se.example_datasets.toy_example(duration=50, num_channels=8, K=5, seed=1)\n",
```

Comment:
This example has been greatly simplified because we can now create a LabboxEphysRecordingExtractor/LabboxEphysSortingExtractor from non-le extractors.